impl/src/main/java/org/ehcache/impl/internal/store/offheap/AbstractOffHeapStore.java

```diff
@@ -1200,11 +1201,11 @@ public Boolean apply() {
     }
   }
 
-  public void handleOversizeMappingException(K key, OversizeMappingException cause, StoreEventSink<K, V> eventSink) throws StoreAccessException {
+  void handleOversizeMappingException(K key, OversizeMappingException cause, StoreEventSink<K, V> eventSink) throws StoreAccessException {
```

Comment:
Not really, that came with the Ehcache 2.x port and was never looked at. In this context, I looked at it and this public is wrong.